src/makeRobot.js

```diff
@@ -38,7 +38,64 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name: name,
+    wheels: wheels,
+    version: version,
```

Comment:
it can be set like that:
```suggestion
    name,
    wheels,
    version,
```
checklist point 1